SU2_CFD/src/solvers/CNSSolver.cpp

```diff
@@ -613,6 +613,144 @@ su2double CNSSolver::GetCHTWallTemperature(const CConfig* config, unsigned short
   return Twall;
 }
 
+void CNSSolver::BC_RadiativeEquilibrium_Wall(CGeometry *geometry,
+                                             CSolver **solver_container,
+                                             CNumerics *conv_numerics,
+                                             CNumerics *visc_numerics,
+                                             CConfig *config,
+                                             unsigned short val_marker){
+    
```

Comment:
It can still have it's own function but that would simply call either BC_Isothermal_Wall_Generic or BC_HeatFlux_Wall_Generic (whichever looks closer to what you want, heat flux I suppose).